### ISSUE TITLE
the performance of your released occupancy model tpv04_occupancy

### ISSUE THREAD

**Issue by SISTMrL** [wzzheng/TPVFormer#39]
![image](https://github.com/wzzheng/TPVFormer/assets/47938053/714e194f-71aa-4f2c-888b-69910513039f)
hello, i run the inference of tpv04_occupancy with command, the gpu i used is v100

python eval.py --py-config config/tpv04_occupancy.py --ckpts ckpt/tpv04_occupancy_v2.pth

the performance on nuscenes is shown in the picture.

the performance is not matched with paper released and could you please explain the evaluation metrics of miou vox/pts, i'm confused about it.

**Comment by huang-yh** [comment]
We use different supervision signals for lidar segmentation and occupancy prediction, and thus yielding different model weights for these two tasks. For lidar segmentation, lovasz softmax loss supervises point predictions and crossentropy loss supervises voxel predictions. While for occupancy prediction, both of them supervise voxel predictions.

**Comment by huang-yh** [comment]
As for pts miou and vox miou, they are not two metrics, but two different ways to derive predictions for point queries. 
Pts miou interpolates voxel features to explicitly generate point features before feeding them to the decoder, while in vox miou, we simply assign the query point the prediction for the voxel which it belongs to.
